### DISCUSSION TITLE
ies render/view

### DISCUSSION THREAD

**Discussion by tmdag** [dlippok/photometric-viewer#115] [Ideas]
This tool is great!

It could be also an awesome tool for 3D artists utilizing .ies files in 3D Renderers.
Currently, when we download .ies file on diskl, there is no good tool to quickly browse and see how this photometric file would look like. Similarly to https://ieslibrary.com/browse would be amazing to be able to quickly browse and see how you .ies files are supposed to look like visually.

**Comment by dlippok** [comment]
Hello Albert and thanks a lot! 

Showing a 3D render of the light source is something I already considered. Unfortunately I haven't found any library that I could use in my project and as I have no experience in 3D programming whatsoever, creating it from scratch would be out of scope for me at the moment.

But I will keep it at the back of my mind and if by chance someone would like to contribute with their code I would be happy to include this feature in the project.

**Reply by vanillajonathan** [reply]
You can do it with a [`Gtk.GLArea`](https://docs.gtk.org/gtk4/class.GLArea.html) widget.
